docs/source/quicktour.rst

```diff
@@ -0,0 +1,372 @@
+    print(batch)
+
+The padding is automatically applied on the side the model expect it (in this case, on the right), with the
+padding token the model was pretrained with. The attention mask is also adapted to take the padding into account:
+
+::
+
+    {'input_ids': tensor([[  101,  2057,  2024,  2200,  3407,  2000,  2265,  2017,  1996, 19081, 3075,  1012,   102],
+                          [  101,  2057,  3246,  2017,  2123,  1005,  1056,  5223,  2009,  1012,  102,     0,     0]]), 
+     'attention_mask': tensor([[1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+                               [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0]])}
+
+You can learn more about tokenizers in this tutorial (coming soon).
+
+Using the model
+^^^^^^^^^^^^^^^
+
+Once your input has been preprocessed by the tokenizer, you can directly send it to the model. As we mentioned, it will
+contain all the relevant information the model needs. If you're using a TensorFlow model, you can directly pass the
+dictionary keys to tensor, for a PyTorch model, you need to unpack the dictionary by adding :obj:`**`.
+
+::
+
+    ## PYTORCH CODE
+    outputs = model(**batch)
+    ## TENSORFLOW CODE
+    outputs = model(batch)
+
+In 🤗 Transformers, all outputs are tuples (with only one element potentially). Here, we get a tuple with just the
+final activations of the model.
+
+::
+
+    (tensor([[-4.1329,  4.3811],
+             [ 0.0818, -0.0418]]),)
+
+.. note::
+
+    All 🤗 Transformers models (PyTorch or TensorFlow) return the activations of the model *before* the final
+    activation function (like SoftMax) since this final activation function is often fused with the loss.
+
+Let's apply the SoftMax activation to get predictions.
+
+::
+
+    ## PYTORCH CODE
+    import torch.nn.functional as F
+    predictions = F.softmax(outputs[0], dim=-1)
+    print(predictions)
+    ## TENSORFLOW CODE
+    predictions = tf.nn.softmax(outputs[0], axis=-1)
+    print(predictions)
+
+We can see we get the numbers from before:
+
+::
+
+    tensor([[2.0060e-04, 9.9980e-01],
+            [5.3086e-01, 4.6914e-01]])
+
+If you have labels, you can provide them to the model, it will return a tuple with the loss and the final activations.
+
+::
+
+    ## PYTORCH CODE
+    import torch
+    outputs = model(**batch, labels = torch.tensor([1, 0])
+    ## TENSORFLOW CODE
+    import tensorflow as tf
+    outputs = model(batch, labels = tf.constant([1, 0])
+
+Models are standard `torch.nn.Module <https://pytorch.org/docs/stable/nn.html#torch.nn.Module>`__ or
+`tf.keras.Model <https://www.tensorflow.org/api_docs/python/tf/keras/Model>`__ so you can use them in your usual
+training loop. 🤗 Transformers also provides a :class:`~transformers.Trainer` (or :class:`~transformers.TFTrainer` if
+you are using TensorFlow) class to help with your training (taking care of things such as distributed training, mixed
+precision, etc.). See the training tutorial (coming soon) for more details.

```

Comment:
That's @joeddav tutorial in #5034

Reply:
We should have more easy to use training tutorials, especially for seq2seq models. There are sooo many questions on github asking for T5, Bart and EncoderDecoder training. I added some sample code to the T5 code page, but I like @joeddav approach of having a unified training page where we should at least for `MaskedLM`, `CausalLM` and `Seq2Seq` models how to train in like 10 lines and on top of that how to use the trainer. 

Also at the moment it is not possible to use TFTrainer with `BertForMaskedLM`, `BertForCausalLM` since the forward function does not yet accept a `labels` parameter. I'd be happy to work with @jplu on this in a couple of days :-)

Reply:
I will be happy to work with you on making the `TFTrainer` compliant for language modeling :)